addons/calendar/models/calendar.py

```diff
@@ -170,7 +170,7 @@ def _send_mail_to_attendees(self, template_xmlid, force_send=False, force_event_
             return res
 
         calendar_view = self.env.ref('calendar.view_calendar_event_calendar')
-        invitation_template = self.env.ref(template_xmlid)
+        invitation_template = template_id
```

Comment:
I think we can stick on template parameter through the whole method and remove "invitation_template" variable that does not mean anything anymore. Indeed thanks to your task template is now generic.